### PR TITLE
added quotas and limitations tables

### PR DESCRIPTION
**Pull request by KanerLev**

### CODE REVIEW

**Comment by Pratichhya** [comment]
@KanerLev some comments:

1. I saw you have mentioned Keycloak and Access tokens as well. Please make sure of the name since a couple of discussions were ongoing with it. #192 
2. For the section "The following limitations and quotas are monthly and reset on the 1st of every month:" I would suggest not to give numbering and using bullets instead to avoid number confusion with the refernce notes above.
3. If you don't want to include in this PR, maybe create a issue for better visualization so that it could be followed.

File: Quotas.qmd [hunk 1]
```diff
@@ -0,0 +1,643 @@
...
+  The following limitations and quotas are monthy and reset on the first day of every month:<br><br>
+  Requests per month<br>
+  Requests per minute<br>
+  Processing units (PU) per month<br>
+  Processing units (PU) per minute<br>
+  Monthly transfer limit (TB) (IAD¹)¹⁰<br>
+  Number of concurrent orders limit (DAD²)<br>
+  Monthly transfer limit (TB) (DAD²)<br>
+  Processed Products per month<br>
+  Credits per month<br><br>
```
Comment (Pratichhya) [inline Quotas.qmd:160]: Thank you for removing the numbering but I also suggested using bullets instead. What do you think of that?

File: Quotas.qmd [hunk 2]
```diff
@@ -0,0 +1,643 @@
...
+<div class="table-responsive">
+<table class="table-roadmap">
+  <tr>
+    <th style="width:6%">#</th>
+    <th>Limitations/Quotas</th>
+    <th>Sentinel Hub APIs⁸</th>
+    <th>Data Workspace API</th>
+    <th>OData, STAC, S3</th>
+    <th>Direct HTTP access to COGs</th>
+    <th>openEO API / Algorithm plaza</th>
```
Comment (Pratichhya) [inline Quotas.qmd:18]: As discussed last time, I think we should show a distinction of a header with the rest of the table. 

Though I would still prefer the CDSE theme, if you think it is not necessary here, I would still suggest including either a horizontal line after header and making headers bold

File: Quotas.qmd (hunk 1 above)
Comment (KanerLev) [reply Quotas.qmd:160]: Sure. Oh I misunderstood that. Sound good, I'm implementing bullets in my upcoming commit.

File: Quotas.qmd (hunk 2 above)
Comment (KanerLev) [reply Quotas.qmd:18]: OK then I also prefer applying the usual CDSE theme, I'll implement it in my upcoming commit.

**Review by Pratichhya — APPROVED** [review]

**Pull request merged**